src/generated/todo/getTodosSaga.js

```diff
@@ -0,0 +1,75 @@
+// This file is generated
+
+import { takeEvery } from 'redux-saga';
```

Comment:
Can we assume that we will always use `takeEvery`? Are there any cases where we'd prefer to `takeLatest` instead?